appengine/flexible_python37_and_earlier/README.md

```diff
@@ -0,0 +1,73 @@
+## Google App Engine Flexible Environment Python Samples
+
+[![Open in Cloud Shell][shell_img]][shell_link]
+
+[shell_img]: http://gstatic.com/cloudssh/images/open-btn.png
+[shell_link]: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=appengine/flexible/README.md
+
+These are samples for using Python on Google App Engine Flexible Environment. These samples are typically referenced from the [docs](https://cloud.google.com/appengine/docs).
+
+See our other [Google Cloud Platform github repos](https://github.com/GoogleCloudPlatform) for sample applications and
+scaffolding for other frameworks and use cases.
+
+## Run Locally
+
+Some samples have specific instructions. If there is a README in the sample folder, please refer to it for any additional steps required to run the sample.
+
+In general, the samples typically require:
+
+1. Install the [Google Cloud SDK](https://cloud.google.com/sdk/), including the [gcloud tool](https://cloud.google.com/sdk/gcloud/), and [gcloud app component](https://cloud.google.com/sdk/gcloud-app).
+
+2. Setup the gcloud tool. This provides authentication to Google Cloud APIs and services.
+
+   ```
+   gcloud init
+   ```
+
+3. Clone this repo.
+
+   ```
+   git clone https://github.com/GoogleCloudPlatform/python-docs-samples.git
+   cd python-docs-samples/appengine/flexible
```

Comment:
```suggestion
   cd python-docs-samples/appengine/flexible_python37_and_earlier
```

Reply:
Good catch! I will update all of them together.